eventbridge-pipes-amazonmq-to-eventbridge-sam/template.yaml

```diff
@@ -0,0 +1,255 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: 'Serverless patterns - Amazon MQ in a Private VPC to Amazon EventBridge Message Bus using Amazon EventBridge Pipes'
+
+Parameters:
+  MQSecretARN:
+    Type: String
+    Description: 'Enter the ARN of the AWS Secrets Manager secret holding a username and password for the Amazon MQ cluster.'
+  MQVpcId: 
+    Type: String
+    Description: 'Enter the Amazon Virtual Private Cloud (VPC) ID to host the Amazon MQ instance.'
+  MQSubnetId:  
+    Type: String
+    Description: 'Enter the Subnet Id to host the Amazon MQ instance. This subnet requires a route to the Public Internet'
```

Comment:
Is it recommended to deploy Amazon MQ to subnet that has public internet access? 

Since EvB does not support VPC Endpoints as of today, is NAT Gateway route recommended?

Reply:
I adjusted the `Description` to include a link to the EB [documentation](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-pipes-mq.html#pipes-mq-vpc-config). The subnet has to have a route to the public internet which could be via NAT GW. Hopefully this adjustment makes the wording clearer.